package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/cmc-common-frontend",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
that would be for a bug fix. the middle one is for a feature change:

New features which don't break existing features: Minor release, increment the middle number, e.g. 1.1.0

https://docs.npmjs.com/getting-started/semantic-versioning